RecoLocalCalo/EcalRecAlgos/src/PulseChiSqSNNLS.cc

```diff
@@ -79,51 +79,82 @@ PulseChiSqSNNLS::PulseChiSqSNNLS() :
   _maxiterwarnings(true)
 {
   
-  Eigen::initParallel();
```

Comment:
https://eigen.tuxfamily.org/dox/TopicMultiThreading.html

_In the case your own application is multithreaded, and multiple threads make calls to Eigen, then you have to initialize Eigen by calling the following routine before creating the threads_

Is removal justified?